crates/apub_objects/src/objects/comment.rs

```diff
@@ -229,6 +229,7 @@ impl Object for ApubComment {
       local: Some(false),
       language_id,
       federation_pending: Some(false),
+      locked: Some(false),
```

Comment:
I know its the same but `None` is probably better

Reply:
Definitely needs to be `None` so it doesnt unlock a comment that was previously locked with LockPageOrNote activity.

Reply:
Also in this file you need to check for incoming federated comments if the parent comment is locked (just like in the api).